lib/ci/container.rb

```diff
@@ -62,29 +60,22 @@ def start(options = {})
       end
     end
 
-    # Default container start/run arguments.
-    # - Binds: Set to binds used on {create}
-    # - Ulimits: Set to sane defaults with lower nofile property
-    # @return [Hash]
-    def default_start_options
-      @default_start_options ||= {
-        Binds: DirectBindingArray.to_bindings(@binds),
-        # Force standard ulimit in the container.
-        # Otherwise pretty much all APT IO operations are insanely slow:
-        # https://bugs.launchpad.net/ubuntu/+source/apt/+bug/1332440
-        # This in particular affects apt-extracttemplates which will take up to
-        # 20 minutes where it should take maybe 1/10 of that.
-        Ulimits: [{ Name: 'nofile', Soft: 1024, Hard: 1024 }]
-      }
-    end
-
     class << self
       # Default container create arguments.
       # - WorkingDir: Set to Dir.pwd
       # - Env: Sensible defaults for LANG, PATH, DEBIAN_FRONTEND
+      # - Binds: Set to binds used on {create}
+      # - Ulimits: Set to sane defaults with lower nofile property
       # @return [Hash]
       def default_create_options
         {
+          # Force standard ulimit in the container.
+          # Otherwise pretty much all APT IO operations are insanely slow:
+          # https://bugs.launchpad.net/ubuntu/+source/apt/+bug/1332440
+          # This in particular affects apt-extracttemplates which will take up
+          # to 20 minutes where it should take maybe 1/10 of that.
+          Ulimits: [{ Name: 'nofile', Soft: 1024, Hard: 1024 }],
+          Binds: DirectBindingArray.to_bindings(@binds),
```

Comment:
/tmp/pangea-tooling/lib/ci/container.rb:78: warning: instance variable @binds not initialized